primefaces/src/main/resources/META-INF/resources/primefaces/wizard/wizard.js

```diff
@@ -70,6 +70,9 @@ PrimeFaces.widget.Wizard = PrimeFaces.widget.BaseWidget.extend({
             //events
             this.backNav.on("click", function() {_self.back();});
             this.nextNav.on("click", function() {_self.next();});
+            if (this.cfg.disableOnAjax) {
+                this.bindTriggers();
```

Comment:
I am pretty sure because you default to true and the renderer says true is default it won't send the value in JS. So this value will never be true. In other widgets you will see we do if undefined make the value true.  This way it saves network traffic.

Reply:
Oh is that why the code I stole this from did an explicit `!== false`?